data/Makefile.am

```diff
@@ -39,8 +39,34 @@ install-sample-data:
 	rm fedora-20-icons.tar.gz; \
 	cd -
 
+com.endlessm.SystemExternalApps.rules: com.endlessm.SystemExternalApps.rules.in
```

Comment:
I honestly feel like this is not gnome-software specific at all since every part of it is flatpak specific.